redfish/resource_redfish_storage_volume.go

```diff
@@ -9,17 +9,19 @@ import (
 	"github.com/stmcginnis/gofish"
 	redfishcommon "github.com/stmcginnis/gofish/common"
 	"github.com/stmcginnis/gofish/redfish"
+	_ "log"
 	"net/http"
 )
 
 const (
 	// This constants are used to avoid hardcoding the terraform input variables
-	storageControllerID string = "storage_controller_id"
-	volumeName          string = "volume_name"
-	volumeType          string = "volume_type"
-	volumeDisks         string = "volume_disks"
-	settingsApplyTime   string = "settings_apply_time"
-	biosConfigJobURI    string = "bios_config_job_uri"
+	storageControllerID   string = "storage_controller_id"
```

Comment:
Using constants might lead to additional work in future for deprecation if these attributes' name changes. I guess we should continue to follow the same guidelines as other terraform providers and use the attribute names directly in schema definition.